qiskit/transpiler/passes/layout/dense_layout.py

```diff
@@ -115,18 +116,20 @@ def run(self, dag):
                 map_iter += 1
         self.property_set['layout'] = layout
 
-    def _best_subset(self, n_qubits):
+    @deprecate_arguments({'n_qubits': 'num_qubits'})
```

Comment:
You don't need to bother with this here, it's a private method in a pass. If someone is using this outside of dense_layout they can not expect this to have a stable interface, especially if they're passing in the sole positional argument by name. Lets just swap it out and remove all the deprecation.